### DISCUSSION TITLE
My moto e 2014 bricked

### DISCUSSION THREAD

**Discussion by vizPilot** [aravindvnair99/Motorola-Moto-E-XT1022-condor-unbrick#6] [Q&A] [answered]
My moto e 2014 bricked in edl mode

it all started when my motorola moto e 2014 was good and one day I installed the sbl1 partition by mistake and it bricked me please help me![IMG_20210320_152758_448.jpg](https://user-images.githubusercontent.com/66806243/111883730-e7fd3780-899b-11eb-84d2-866a84501733.jpg)

**Comment by aravindvnair99** [comment]
@Vicc2008 Have you tried the steps that are already mentioned?

**Comment by vizPilot** [comment]
But i wanted to install Windows phone 8.1 to my moto e 2014

**Comment by vizPilot** [comment]
> @Vicc2008 Have you tried the steps that are already mentioned?

Yes but i get errors

**Comment by aravindvnair99** [comment]
> > @Vicc2008 Have you tried the steps that are already mentioned?
> 
> Yes but i get errors

@Vicc2008 Make sure you have met all prerequisites and requirements.

**Comment by vizPilot** [comment]
In my case is a qualcomm snapdragon msm8210 and Android 5.1 Lollipop

**Comment by vizPilot** [comment]
But i wanted to install Windows phone 8.1 to my moto e 2014

**Comment by aravindvnair99** [comment, marked as answer]
> But i wanted to install Windows phone 8.1 to my moto e 2014

@Vicc2008 If your phone is bricked, you need to unbrick it first. Also, I no longer have a Moto E with me to experiment and this project is no longer maintained. You might want to check the following which is what worked years ago for many people:

- https://github.com/aravindvnair99/Motorola-Moto-E-XT1022-condor-unbrick/blob/main/Prerequisite.md
- https://github.com/aravindvnair99/Motorola-Moto-E-XT1022-condor-unbrick/blob/main/Unbrick%20methods.md
- https://github.com/aravindvnair99/Motorola-Moto-E-XT1022-condor-unbrick/tree/main/Proofs

<blockquote><img src="https://avatars.githubusercontent.com/u/22199259?s=400&v=4" width="48" align="right"><div><img src="https://github.githubassets.com/favicons/favicon.svg" height="14"> GitHub</div><div><strong><a href="https://github.com/aravindvnair99/Motorola-Moto-E-XT1022-condor-unbrick">aravindvnair99/Motorola-Moto-E-XT1022-condor-unbrick</a></strong></div><div>Tools for Motorola Moto E XT1022, XT1021, XT1025 condor unbrick - aravindvnair99/Motorola-Moto-E-XT1022-condor-unbrick</div></blockquote>
<blockquote><img src="https://avatars.githubusercontent.com/u/22199259?s=400&v=4" width="48" align="right"><div><img src="https://github.githubassets.com/favicons/favicon.svg" height="14"> GitHub</div><div><strong><a href="https://github.com/aravindvnair99/Motorola-Moto-E-XT1022-condor-unbrick">aravindvnair99/Motorola-Moto-E-XT1022-condor-unbrick</a></strong></div><div>Tools for Motorola Moto E XT1022, XT1021, XT1025 condor unbrick - aravindvnair99/Motorola-Moto-E-XT1022-condor-unbrick</div></blockquote>
<blockquote><img src="https://avatars.githubusercontent.com/u/22199259?s=400&v=4" width="48" align="right"><div><img src="https://github.githubassets.com/favicons/favicon.svg" height="14"> GitHub</div><div><strong><a href="https://github.com/aravindvnair99/Motorola-Moto-E-XT1022-condor-unbrick">aravindvnair99/Motorola-Moto-E-XT1022-condor-unbrick</a></strong></div><div>Tools for Motorola Moto E XT1022, XT1021, XT1025 condor unbrick - aravindvnair99/Motorola-Moto-E-XT1022-condor-unbrick</div></blockquote>

**Comment by vizPilot** [comment]
In my case my model is a XT1021

**Comment by aravindvnair99** [comment]
> In my case my model is a XT1021

@Vicc2008 Works on XT1022, XT1021 and XT1025 as mentioned [here](https://github.com/aravindvnair99/Motorola-Moto-E-XT1022-condor-unbrick/blob/main/README.md#motorola-moto-e-xt1022-condor-unbrick-).

**Comment by vizPilot** [comment]
I need a blank-flash for the xt1021 but with the UEFI

**Comment by aravindvnair99** [comment]
> I need a blank-flash for the xt1021 but with the UEFI

@Vicc2008 I'm sorry, I no longer have a Moto E with me nor am I maintaining or supporting this project any longer. The Moto E is an old phone now. This was like a hobby for me 4 years ago or so and not anymore. You are free to go through the stuff I have put up here and XDA, the links which I have given you already. But beyond that, I don't have anything to say.

**Comment by vizPilot** [comment]
Oh

**Comment by vizPilot** [comment]
or with thor2 from Windows device recovery tool

**Comment by aravindvnair99** [comment]
> or with thor2 from Windows device recovery tool

@Vicc2008 The `Windows Device Recovery Tool` is not meant for Android devices.

**Comment by vizPilot** [comment]
Oh

**Comment by vizPilot** [comment]
but from cmd

**Comment by aravindvnair99** [comment]
> but from cmd

@Vicc2008 CMD method is here: https://github.com/aravindvnair99/Motorola-Moto-E-XT1022-condor-unbrick/blob/main/Unbrick%20methods.md#method-5-fastboot

<blockquote><img src="https://avatars.githubusercontent.com/u/22199259?s=400&v=4" width="48" align="right"><div><img src="https://github.githubassets.com/favicons/favicon.svg" height="14"> GitHub</div><div><strong><a href="https://github.com/aravindvnair99/Motorola-Moto-E-XT1022-condor-unbrick">aravindvnair99/Motorola-Moto-E-XT1022-condor-unbrick</a></strong></div><div>Tools for Motorola Moto E XT1022, XT1021, XT1025 condor unbrick - aravindvnair99/Motorola-Moto-E-XT1022-condor-unbrick</div></blockquote>

**Comment by vizPilot** [comment]
But in edl mode

**Comment by vizPilot** [comment]
With blank-flash

**Comment by aravindvnair99** [comment]
> With blank-flash

@Vicc2008 https://github.com/aravindvnair99/Motorola-Moto-E-XT1022-condor-unbrick/blob/main/Unbrick%20methods.md#method-2-blank-flash

<blockquote><img src="https://avatars.githubusercontent.com/u/22199259?s=400&v=4" width="48" align="right"><div><img src="https://github.githubassets.com/favicons/favicon.svg" height="14"> GitHub</div><div><strong><a href="https://github.com/aravindvnair99/Motorola-Moto-E-XT1022-condor-unbrick">aravindvnair99/Motorola-Moto-E-XT1022-condor-unbrick</a></strong></div><div>Tools for Motorola Moto E XT1022, XT1021, XT1025 condor unbrick - aravindvnair99/Motorola-Moto-E-XT1022-condor-unbrick</div></blockquote>

**Comment by vizPilot** [comment]
Oh
